mkosi/qemu.py

```diff
@@ -144,11 +144,17 @@ def identify(cls, config: Config, path: Path) -> "KernelType":
             logging.warning("bootctl doesn't know kernel-identify verb, assuming 'unknown' kernel type")
             return KernelType.unknown
 
-        type = run(
-            ["bootctl", "kernel-identify", path],
-            stdout=subprocess.PIPE,
-            sandbox=config.sandbox(options=["--ro-bind", path, path]),
-        ).stdout.strip()
+        try:
+            type = run(
+                ["bootctl", "kernel-identify", path],
+                stdout=subprocess.PIPE,
+                sandbox=config.sandbox(options=["--ro-bind", path, path]),
+            ).stdout.strip()
+        except subprocess. CalledProcessError as e:
+            # If an error occurs, set a default value for 'type'
+            logging.warning(f"'bootctl kernel-identify' returned a exit code {e.returncode} for '{path}'")
+            logging.warning(f"'assuming type 'unknown'")
```

Comment:
Instead of logging to separate lines
```suggestion
            logging.warning(
                f"'bootctl kernel-identify' returned a exit code {e.returncode} for '{path}' assuming type 'unknown'"
            )
```